protocol/bc/value.go

```diff
@@ -4,4 +4,8 @@ type ValueSource struct {
 	Ref      Hash
 	Value    AssetAmount
 	Position uint64 // what int do we actually want?
+
+	// The Entry corresponding to Ref, if available
```

Comment:
Are there any rules for when this is or isn't set? It would be comforting to know what to expect.